2019-Feburary-26/writingSamples/tong.tex

```diff
@@ -0,0 +1,165 @@
+regression method. I have cut out many details such as description
+of/steps involved in deep learning, as I will mention them in the Method
+section. Please let me know if I need to further explain anything and if
+my justification is lacking
+
+I have grayed out the portion that exceeds the word limit, which I
+included because I think it provides some context, i.e., how I think my
+study can address the unknown in the literature. I would appreciate your
+comments on this portion, but you don't have to.
+
+Thank you!
+
+\textbf{3.2. Strengths of the Current Study}
+
+In the extant literature, no quantitative meta-analysis has been run to
+find consistent findings of both brain morphometry and connectivity
+alterations across studies, and a qualitative review paper (McPhee et al
+2018) pointed out inconsistent findings on brain morphometry changes,
+even for the most robust effect (reduced amygdalar volume). In addition,
+previous studies on both brain morphometry and resting-state functional
+connectivity had relatively small sample size (less than 100 FHP, except
+for Dager et al 2015). In many studies, FHP had substance use history,
+making it difficult to tease apart neural alterations associated FH
+status and personal substance use.
+
+The Adolescent Brain Cognitive Development study provides a rich dataset
+from more than 11,000 children age 9-10 with minimal substance use.
+Using deep learning to analyze data from substance naïve FHN and FHP
+children from this dataset can elucidate changes in brain structure and
+connectivity associated with FH of AUD in a large group of participants.
+
+\textbf{3.2.1 Strengths of machine learning}
+
+Traditional whole brain analysis for both structural and connectivity
+data typically involves mass-univariate analysis, which has significant
+limitations (Vieira et al 2017). Firstly, mass-univariate analysis
+involves multiple independent comparisons (e.g., independently comparing
+amygdalar and hippocampal volumes between FHP and FHN). This results in
+stringent correction for multiple comparisons, which can increase Type
+II error. In addition, multiple comparisons assume that each brain
+regions are independent, which is not accurate. Vieira et al (2017) also
+noted that mass-univariate techniques can have statistical inferences at
+the group level, but not at the level of the individual. Diagnostic and
+treatment decisions, on the other hand, are made at the individual
+level.
+
+These limitations can be addressed with machine learning, which is a
+multivariate method to detect trends and patterns in existing data,
+which can later be used to make predictions on new data. Machine
+learning also allows for prediction to be made at the individual level.
+
+The gap in the literature that this project hopes to address is the
+inconsistent findings in brain structure and functional connectivity in
+FH of AUD. Using a large sample from the ABCD database is one of the
```

Comment:
is the fact that ABCD is a large sample covered in the previous section? is this section only focusing on machine learning?